code/datums/gamemodes/spy_theft.dm

```diff
@@ -341,151 +341,172 @@
-		big_station_bounties += list(list(/obj/machinery/communications_dish, 2))
-		big_station_bounties += list(list(/obj/decal/poster/wallsign/poster_y4nt, 1))
+		big_station_bounties[/obj/machinery/vehicle] = 1
+		big_station_bounties[/obj/machinery/chem_dispenser] = 2
+		big_station_bounties[/obj/machinery/computer/announcement] = 2
+		big_station_bounties[/obj/machinery/computer/card] = 2
+		big_station_bounties[/obj/machinery/computer/aiupload] = 3
+		big_station_bounties[/obj/machinery/computer/genetics] = 2
+		big_station_bounties[/obj/machinery/computer/supplycomp] = 1
+		big_station_bounties[/obj/machinery/computer/robotics] = 2
+		big_station_bounties[/obj/machinery/power/reactor_stats] = 1
+
+		big_station_bounties[/obj/machinery/computer3/generic/communications] = 1
+		big_station_bounties[/obj/machinery/computer3/terminal/zeta] = 1
+		big_station_bounties[/obj/machinery/vending/security] = 2
+		big_station_bounties[/obj/machinery/vending/medical] = 1
+		big_station_bounties[/obj/machinery/vending/port_a_nanomed] = 1
+		big_station_bounties[/obj/machinery/vending/fortune] = 1
+		big_station_bounties[/obj/machinery/vending/standard] = 1
+
+		big_station_bounties[/obj/machinery/port_a_brig] = 3
+		big_station_bounties[/obj/machinery/flasher/portable] = 2
+		big_station_bounties[/obj/machinery/sleeper/port_a_medbay] = 1
+		big_station_bounties[/obj/machinery/atmospherics/unary/cryo_cell] = 2
+		big_station_bounties[/obj/machinery/computer/cloning] = 2
+		big_station_bounties[/obj/machinery/clonepod] = 2
+		big_station_bounties[/obj/machinery/clonegrinder] = 1
+		big_station_bounties[/obj/machinery/genetics_scanner] = 2
+
+		big_station_bounties[/obj/machinery/recharge_station] = 1
+		big_station_bounties[/obj/machinery/optable] = 2
+		big_station_bounties[/obj/storage/closet/port_a_sci] = 2
+
+		big_station_bounties[/obj/machinery/manufacturer/robotics] = 1
+		big_station_bounties[/obj/machinery/manufacturer/medical] = 1
+		big_station_bounties[/obj/machinery/manufacturer/general] = 1
+		big_station_bounties[/obj/machinery/manufacturer/general] = 1
+
+		big_station_bounties[/obj/submachine/chef_oven] = 1
+		big_station_bounties[/obj/machinery/gibber] = 1
+
+		big_station_bounties[/obj/machinery/bot/guardbot] = 1
+		big_station_bounties[/obj/machinery/plantpot] = 1
+		big_station_bounties[/obj/machinery/partyalarm] = 1
+		big_station_bounties[/obj/pool_springboard] = 1
+
+		big_station_bounties[/obj/reagent_dispensers] = 2
+		big_station_bounties[/obj/machinery/crusher] = 2
+		big_station_bounties[/obj/machinery/communications_dish] = 2
 
 	active_bounties.len = 0
 
+	// Find matches for all our possible bounties
+	var/list/spy_thief_target_types = list()
+	// Construct master list of target types
+	for(var/i=1, i<=station_bounties.len, i++)
+		spy_thief_target_types += station_bounties[i]
+	for(var/i=1, i<=big_station_bounties.len, i++)
+		spy_thief_target_types += big_station_bounties[i]
```

Comment:
```suggestion
	var/list/spy_thief_target_types = station_bounties + big_station_bounties
```

Reply:
This should also make Azrun's suggestion of `spy_thief_target_types[Object.type]` work.